website/content/bounties.njk

```diff
@@ -44,7 +44,8 @@ description: The MonoGame Foundation uses a bounty system to propose paid incent
             <span class="fw-bold">Before you apply for fulfilling a bounty, please note the following requirements:</span>
         </p>
 		<ul>
-		  <li>You must be a registered business or a registered self-employed individual.</li>
+		  <li>For US citizens you must provide a <a href="https://www.irs.gov/forms-pubs/about-form-w-9">W-9</a> form along with your signed NDA when selected for a bounty.</li>
+          <li>For non-US citizens you need to provide a <a href="https://www.irs.gov/forms-pubs/about-form-w-8">W-8</a> form along with your signed NDA when selected for a bounty </li>
```

Comment:
We need to maybe say "provide the correct W-8 form" and link the two main ones... the one for individuals and the one for entities (corporations).

That said it is their responsibility to give us the correct version of the W-8.  The individual one will be the correct one 99% of the time.

Reply:
The link I put for W-8 (https://www.irs.gov/forms-pubs/about-form-w-8) is the landing page for ALL the types of from. But I can change the working to "correct"